tests/test_sitl.py

```diff
@@ -39,6 +43,34 @@ def test_version_list():
     models.sort()
     assert_equals(expected, models)
 
+
 def test_download():
     from dronekit_sitl import download
-    download('copter','3.3', None)
\ No newline at end of file
+    download('copter','3.3', None)
+
+
```

Comment:
That would make it inconsistent with the rest of the file.  I'll add a patch on top to make them all single-spaced.